packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/ip-subnet-validation.util.js

```diff
@@ -0,0 +1,44 @@
+// Checks if the IP is in the subnet of CIDR
```

Comment:
would it be possible to use an existing library for that case, for example : https://github.com/whitequark/ipaddr.js

Reply:
I will check this suggestion.

Reply:
Done.